common.yml

```diff
@@ -1801,10 +1834,12 @@ buildvariants:
       - name: "dist"
       - name: "sign"
         run_on: rhel87-small
+      - name: "record-in-papertrail"
```

Comment:
I assume we want to record every release in the papertrail, why not extend the `push` task instead?

Reply:
Yes, good call. I changed this.